### ISSUE TITLE
Request: Query Band Read/Write Support

### ISSUE THREAD

**Issue by r-richmond** [tableau/document-api-python#109]
Certain Datasources have query bands and it would be extremely helpful to be able to modify these programatically.

**Comment by t8y8** [comment]
@r-richmond Thank you for opening a request!

Would it be possible to include a dummy-tds or twb that already has a query band in it? 
It's very helpful for us when building test cases and understand the requirements.

(I'm marking this as an enhancement)

**Comment by r-richmond** [comment]
Sure,

I've attached a dummy workbook that contains both sample query bands & sample initial sql.

As far as an example use case. I would like to be able to write something like 

```
for workbook in folder:
  for datasource in workbook:
    if datasource.query_band is none:
      datasource.query_band = 'TableauReport=<workbookname>';
   elif datasource.query_band does not contain 'TableauReport=<workbookname>':
      datasource.query_band += '\n TableauReport=<workbookname>';
```

note: in this case I'd like <workbookname> to be the special value that you can use in query bands now and not a hardcoded string of the current workbook name.

[bands_initial_dummy_workbook.twb.zip](https://github.com/tableau/document-api-python/files/569824/bands_initial_dummy_workbook.twb.zip)

**Comment by r-richmond** [comment]
Marking this as closed with release 0.6 & the merging of #123
